autoshare-for-twitter.php

```diff
@@ -20,11 +20,58 @@
 }
 
 define( 'AUTOSHARE_FOR_TWITTER', __FILE__ );
-define( 'AUTOSHARE_FOR_TWITTER_VERSION', '2.1.0' );
+define( 'AUTOSHARE_FOR_TWITTER_VERSION', '2.1.1' );
 define( 'AUTOSHARE_FOR_TWITTER_URL', plugin_dir_url( __FILE__ ) );
 define( 'AUTOSHARE_FOR_TWITTER_PATH', plugin_dir_path( __FILE__ ) );
 define( 'AUTOSHARE_FOR_TWITTER_INC', AUTOSHARE_FOR_TWITTER_PATH . 'includes/' );
 
+/**
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @since 2.1.1
+ *
+ * @return string Minimum version required.
+ */
+function minimum_php_requirement() {
```

Comment:
I think we should add a prefix to the function name here as we don't have a namespace added here OR maybe add a namespace in the header. what do you think?

Reply:
Good catch. Thanks @iamdharmesh 
I'll check this one. Mostly will add prefix here if we are not using namespace. ( Adding namespace can be separate ticket so can avoid creating any other unexpected issues/changes here - like test cases might need update if namespace is added )

Reply:
Done: https://github.com/10up/autoshare-for-twitter/pull/258/commits/e30968cc56353acfedc635807b59bd48cdd83254

Added namespace instead prefix, since we are already using namespace in all other files.